File: PYTHON/PROGRAM/Shirt.py
from Clothing import Clothing #MENGAMBIL KELAS ORANG TUA AGAR BISA DIWARISKAN

class Shirt(Clothing):  #PENGISIAN KELAS SHIRT YANG DIWARISI OLEH CLOTHING
    #KONSTRUKTOR
    def __init__(self, idProduct, name, brand, price, size, material, gender, color, sleeve_type): 
        super().__init__(idProduct, name, brand, price, size, material, gender) # OBJEK WARISAN DARI CLASS ORTU
        self.color = color
        self.sleeve_type = sleeve_type
    
    #GETTER DAN SETTER
    def get_color(self):
        return self.color

    def set_color(self, color):
        self.color = color
        
    def get_sleeve_type(self):
        return self.sleeve_type

    def set_sleeve_type(self, sleeve_type):
        self.sleeve_type = sleeve_type

File: PHP/PROGRAM/Shirt.php
<?php
class Shirt extends Clothing {
    private $color; //deklarasi objek - objek class shirt
    private $sleeve_type;

    public function __construct($idProduct, $name, $brand, $price, $size, $material, $gender, $color, $sleeve_type)
    {
        parent::__construct($idProduct, $name, $brand, $price, $size, $material, $gender);  //mewariskan objek - objek dari orang tuanya (class product dan clothing)
        $this->color = $color;
        $this->sleeve_type = $sleeve_type;
    }

    //getter dan setter
    public function get_color(){
        return $this->color;
    }

    public function set_color($color){
        $this->color = $color;
    }

    public function get_sleeve_type(){
        return $this->sleeve_type;
    }

    public function set_sleeve_type($sleeve_type){
        $this->sleeve_type = $sleeve_type;
    }
}

?>